packages/documentation/src/content/docs/admin/liquidity/asset-liquidity.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Asset liquidity
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components'
+import { CodeBlock, LinkOut } from '@interledger/docs-design-system'
+import IdempotencyNote from '/src/partials/liquidity-idempotency.mdx'
+
+Asset liquidity is the amount of value, denominated in a given asset, that Rafiki has available to handle cross-currency (foreign exchange) transactions between you and your peer. Whenever an outgoing payment/incoming payment is in a different asset than the peering relationship, the liquidity of asset accounts will change depending on the FX direction.
+
+You should deposit and withdraw liquidity as necessary, based on your risk tolerance. Rafiki will fail any transaction that would cause an asset's liquidity to fall below zero.
+
+For more information about how Rafiki handles liquidity, see the [Accounting](/overview/concepts/accounting) concepts page and the [low asset liquidity](/integration/requirements/webhook-events#low-asset-liquidity) section of the webhook events page.
+
+## Manage asset liquidity using Rafiki Admin
+
+You can deposit and withdraw asset liquidity through the Rafiki Admin application's [Assets](/admin/admin-user-guide/#edit-asset) screen.
+
+## Manage asset liquidity using the Backend Admin API
+
+<IdempotencyNote />
+
+### Deposit asset liquidity
+
+<Tabs>
+  <TabItem label="Operation">
+    ```graphql
+    mutation DepositAssetLiquidity($input: DepositAssetLiquidityInput!) {
+      depositAssetLiquidity(input: $input) {
+        assetId
+        amount
+        id
+        idempotencyKey
+        success
+      }
+    }
+  ```
+  </TabItem>
+  <TabItem label="Arguments">
+    ```json
+    {
+      "input": {
+        "assetId": "7b8b0f65-896d-4403-b7ba-2e24bf20eb35",
+        "amount": "100",
+        "id": "b97fd85a-126e-42ef-b40d-1a50a70ffa6f",
+        "idempotencyKey": "b97fd85a-126e-42ef-b40d-1a50a70ffa6f",
+      }
+    }
+  ```
+  For more information about this mutation's input object, see [`DepositAssetLiquidityInput`](https://rafiki.dev/apis/graphql/backend/inputobjects/#depositassetliquidityinput). 
+  </TabItem>
+  <TabItem label="Response">
+    When an asset liquidity deposit is successful, `DepositAssetLiquidity` returns `true`.
+
+    ```json
+    {
+      data: {
+        success: true
+      }
+    }
```

Comment:
Yes, I wonder why prettier didn't format this with quotes 🤔

Reply:
Also no quotes in payment-liquidity or two-phase-transfers, just peer 🤷🏻‍♀️